src/stacks-hierarchy/RootStack.tsx

```diff
@@ -63,7 +63,43 @@ export const RootStack = () => {
     const params = parse(path);
     let destination: PartialRoute<any>[] | undefined;
 
-    if (departuresV2Enabled) {
+    if (path.includes('details')) {
```

Comment:
The code looks good in general is would just suggest that:
```
{
          name: 'Departures_DepartureDetailsScreen',
          params: {
            activeItemIndex: 0,
            items: [
              {
                serviceJourneyId: params.serviceJourneyId,
                serviceDate: params.serviceDate,
                date: new Date(),
                fromQuayId: params.quayId,
              },
            ],
          },
```

could be added if details are present, otherwise, the code before that looks similar to the next condition.

Reply:
Is this details stuff applies to both v1 and v2? or is it just a v2 thing? in such cases, the `departuresV2Enabled` should be evaluated.

Reply:
Should be solved in the latest commit